apps/src/lib/script-editor/ScriptEditor.jsx

```diff
@@ -174,6 +181,7 @@ export default class ScriptEditor extends React.Component {
                 style={styles.dropdown}
                 defaultValue={this.props.curriculumUmbrella}
                 ref={select => (this.curriculumUmbrellaSelect = select)}
```

Comment:
While you're in here, I think the `name` and `ref` props on this component aren't being used anywhere, and could be removed.

Reply:
I removed ref, but all the other fields have names so I kept that for consistency.  And maybe it will be useful in the upcoming script editor UI tests?

Reply:
:+1: Sounds good.  Thanks!